addons/strings/fnc_unicodeToChar.sqf

```diff
@@ -0,0 +1,62 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_unicodeToChar
+
+Description:
+    Gets the unicode char of the given unicode number
+    For more information to unicodes visit: https://unicode-table.com/
+
+Parameters:
+    _unicodeNumber: the unicode number <NUMBER>
+
+Returns:
+    given unicode char <STRING>
```

Comment:
"unicode char"? Which of the many possible encodings?

Does this function map from a unicode code point number to a UTF-8 string representation of that code point?